eve/workers/pod-unit-tests/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM centos:7
+
+ARG BUILDBOT_VERSION=0.9.12
+
+ENV LANG=en_US.utf8
+
+WORKDIR /home/eve/workspace
+
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN yum install -y epel-release \
+    && yum install -y gcc \
+    make \
+    python-devel \
+    python-pip \
+    python36 \
+    python36-devel \
+    python36-pip \
+    hg \
+    git \
+    nodejs \
```

Comment:
May be interesting to use a `node:${NODE_VERSION}` image instead, now that this works.